CDCgov/NEDSS-Modernization
# Bug/cnft2 1407/fix quickcondition table columns

## Description

The columns on the QuickCondition look up table were off and overlapping.  This corrects them to match figma document.

## Tickets

* [Jira Ticket](https://cdc-nbs.atlassian.net/jira/software/c/projects/CNFT2/boards/99?assignee=712020%3A591f5469-427d-49e9-87dc-8b4df72a02b4&assignee=63e12176c2b1cb6b347323a8&selectedIssue=CNFT2-1407)
* and [Jira Ticket] (https://cdc-nbs.atlassian.net/jira/software/c/projects/CNFT2/boards/99?assignee=712020%3A591f5469-427d-49e9-87dc-8b4df72a02b4&assignee=63e12176c2b1cb6b347323a8&selectedIssue=CNFT2-1418)

## Steps to verify

1. Navigate to /page-builder/add/page
2. Click "search and add condition(s)"
3. Verify the table columns do not overlap.
4. Verify the checkboxes can be checked
